components/Tabs/Tabs.js

```diff
@@ -1,58 +1,65 @@
 class TabLink {
   constructor(element){
     // assign this.element to the element reference
-    this.element;
+    this.element = element;
     // Get the tab data attribute and save the value here
-    this.tabData = ; 
+    this.tabData = this.element.dataset.tab; 
     // Find all elements with the .card class in index.html that correspond to the tab data attribute
     // If the data is 'all' then select all cards regardless of their data attribute
-    if(){
-      this.cards = ;
+    if(this.element.dataset.tab === "all") {
```

Comment:
Since you've already assigned `this.element.dataset.tab` to `this.tabData`, you can simply use `this.tabData` reference here

Reply:
Another thing, for JS, it's good to use single quotes as opposed to using double quotes for HTML

Reply:
Good to know. I've kind of been going back and forth but I'll just use single quotes from now on 
👍🏼.